release-gap-package

```diff
@@ -143,10 +146,12 @@ while [ x"$1" != x ]; do
     --srcdir ) SRC_DIR="$1"; shift ;;
     --webdir ) WEB_DIR="$1"; shift ;;
     --tmpdir ) TMP_DIR="$1"; shift ;;
+    --update-file ) UPDATE_FILE="$1"; shift ;;
```

Comment:
I am not thrilled by the name --update-file, but I also can't think of  good alternative right now, so... *shrug*

Reply:
I think this name already existed in the old homalg_project version of the release script and I simply used it without thinking about it. Maybe "update-script" or "update-website-data-script" would be better?

Reply:
I think `--update-script` is marginally better. But in the end, it's bikeshedding. If you don't feel changing to update-script is to onerous, do it, otherwise, I'll also merge as-is *shrug*.

Reply:
Done, see the last commit :-)